docs/src/howto-guides/0012_install_and_enable_software_management.md

```diff
@@ -0,0 +1,52 @@
+# Install and enable software managment feature
+Below steps show how to download, install and enable thin-edge software management feature.
+
+## Download and install software management packages on the device
+
+As a prerequisite, install [tedge and tedge_mapper](../howto-guides/002_installation.md) if not installed already. 
+
+The thin-edge software management packages are in repository on GitHub: [thin-edge.io](https://github.com/thin-edge/thin-edge.io/releases).
+
+To download the package from github repository use the following command (use desired version):
+
+```shell
+curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/<package>_<version>_<arch>.deb
+```
+
+where:
+> `version` -> thin-edge.io software management components version in x.x.x format
+>
+> `arch` -> architecture type (amd64, armhf)
+
+Download `tedge_apt_plugin` and `tedge_agent`
+
+```shell
+curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/0.3.0/tedge_apt_plugin_0.3.0_amd64.deb
+curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/0.3.0/tedge_agent_0.3.0_amd64.deb
+```
+
+Once the packages are downloaded, proceed to installation.
+> Note: Some OSes may require you to use `sudo` to install packages and therefore all following commands may need `sudo`.
```

Comment:
Most of the OSes require privileged access to install packages. So, I would remove that note and explicitly use `sudo` on all the examples.

Reply:
fixed